### PR TITLE
Various minor fixes

### PR DESCRIPTION
**Pull request by ypid**

### CODE REVIEW

File: ansible/roles/debops-contrib.kodi/tasks/main.yml [hunk 1]
```diff
@@ -30,3 +30,18 @@
...
+    owner: 'root'
+    group: 'root'
+    mode: '0644'
+  when: (kodi__polkit_action != "")
+
+- name: Remove polkit configuration
+  file:
+    path: '/etc/polkit-1/localauthority/50-local.d/kodi-actions.pkla'
+    state: 'absent'
+  when: (kodi__polkit_action == "")
```
Comment (drybjed) [inline ansible/roles/debops-contrib.kodi/tasks/main.yml]: You could try to replace that with:
```yaml
when: not kodi__polkit_action|d()
```
It should have the same result.

Comment (ypid) [reply ansible/roles/debops-contrib.kodi/tasks/main.yml]: Thanks for the hint. Fixed.

**Review by drybjed — APPROVED** [review]
Looks good, thanks for fixing so many misspellings. :-)

**Comment by ypid** [comment]
You are welcome ;-)

**Pull request merged**